packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -134,16 +135,13 @@ abstract contract FundingRateApplier is EmergencyShutdownable, FeePayer {
         nonReentrant()
         returns (FixedPoint.Unsigned memory totalBond)
     {
-        require(fundingRate.proposalTime == 0, "Proposal in progress");
+        require(fundingRate.proposalTime == 0);
```

Comment:
were these stripped out to save bytecode?

Reply:
yes unfortunately.